nondex-instrumentation/src/main/java/edu/illinois/nondex/instr/Instrumenter.java

```diff
@@ -147,31 +216,17 @@ private void process(String rtJar, String outJar)
     }
 
 
-    private void instrumentStandardClasses(ZipFile rt, ZipOutputStream outZip)
+    private void instrumentStandardClasses(ZipOutputStream outZip)
             throws IOException, NoSuchAlgorithmException {
         for (String cl : this.standardClassesToInstrument) {
-            InputStream clInputStream = null;
-            try {
-                ZipEntry entry = rt.getEntry(cl);
-                if (entry == null) {
-                    Logger.getGlobal().log(Level.WARNING, "Could not find " + cl + " in rt.jar");
-                    Logger.getGlobal().log(Level.WARNING, "Are you running java 8?");
-                    Logger.getGlobal().log(Level.WARNING, "Continuing without instrumenting: " + cl);
-
-                    continue;
-                }
-                clInputStream = rt.getInputStream(entry);
-            } catch (IOException exc) {
-                // I am not sure this code is reachable
-                // TODO(gyori): Test that this code is reachable;
-                Logger.getGlobal().log(Level.WARNING, "Cannot find " + cl + " are you sure this is a valid rt.jar?");
-                Logger.getGlobal().log(Level.WARNING, "Continuing without insturmenting: " + cl);
+            InputStream clInputStream = this.getClassInputStream(cl);
+            if (clInputStream == null) {
                 continue;
             }
 
             this.writeMd5(clInputStream, cl + ".md5", outZip);
 
-            clInputStream = rt.getInputStream(rt.getEntry(cl));
+            clInputStream = this.getClassInputStream(cl);
```

Comment:
Is it necessary to do this call again, seeing that it is assigned this way at the beginning of the loop?